suse_migration_services/suse_product.py

```diff
@@ -117,3 +117,24 @@ def get_tag(self, tag):
                 'Parsing XML file {0} failed with: {1}'
                 .format(self.base_product, issue)
             )
+
+    def get_product_name(self):
+        """Get the product name to be migrated to."""
+        migration_product_name = None
+        try:
+            name = self.get_tag('name')[0]
+            arch = self.get_tag('arch')[0]
+            if name and arch:
+                migration_product_name = '/'.join(
+                    [name, str(self.get_default_target_version()), arch]
+                )
+        except Exception as issue:
+            log.error(
+                'No auto detection of baseproduct: {0}. {1}'
+                .format(issue, 'Continue without auto detected product')
+            )
```

Comment:
I think the log message could be misleading. The reaction on what happens if get_product_name() couldn't find the product is not defined. It depends on the code that handles this result. Thus writing a message that says "... we continue without auto detected product" could be just wrong. I think the error message should be just saying what happened. For example:

```
log.error(
    'Base product could not be detected: {0}'.format(issue)
)
```

We also could write an information that the user can specify the product in the custom config file as a plus.
Does this makes sense ?